docs/events/timeline-ui-overview.asciidoc

```diff
@@ -157,8 +157,36 @@ then select _Export selected_.
 * To export multiple Timelines, select all the required Timelines and then click
 *Bulk actions* -> _Export selected_.
 
-. To import Timelines, click *Import Timeline* and then select or drap-and-drop
+. To import Timelines, click *Import Timeline* and then select or drag-and-drop
 the Timeline `ndjson` file.
 +
 NOTE: Each Timeline object in the file must be represented in a single line.
 Multiple Timeline objects are delimited with newlines.
+
+[discrete]
+[[timeline-accessibility-features]]
+== Timeline accessibility features
+
+Accessibility features, such as keyboard focus and screen reader support, are built into the Timeline interface. These features offer additional ways for you to navigate through the interface and interact with draggable and static elements in the Timeline.
+
+[discrete]
+[[draggable-timeline-elements]]
+=== Interacting with draggable timeline elements
+
+From the Timeline tab, use your keyboard to do one of the following:
+
+
+* Select a draggable element on the Timeline to give it keyboard focus.
+* Press *Enter* on an element with keyboard focus to display its menu.
+* Use the spacebar to drag-and-drop an element with keyboard focus to a different location.
+
+
+[discrete]
+[[timeline-tab]]
+=== Navigating the Timeline Tab
```

Comment:
Consider changing

> Navigating the Timeline Tab

to something like

`Navigating Timeline's table`

to avoid confusion with the similarly-named `Timelines` tab.